app/Model/Server.php

```diff
@@ -4310,7 +4310,7 @@ public function dbSchemaDiagnostic()
      */
     private function __attachRecoveryQuery($field, $table)
     {
-        if ($field['is_critical']) {
+        if (isset($field['error_type'])) {
```

Comment:
Do we want to provide the scripts even if we consider them unnecessary? I am not sure either way, just thinking of cases where the changes are actually beneficial (such as the action field length being set to 80 instead of 20 on the log table).

Reply:
IMO, I would like my MISP database to be as close as possible to the excpected SQL schema. it cost absolutely nothing to provide a button with a quick fix.
That's why we have 2 alerts type:
- Error: The schema should be fixed (missing table/column)
- Warning: The schema can be fixed (NULL)

Reply:
A quick fix is not always a quick fix. Again, an alter table for fixing a NULL/NOT NULL flag that is ultimately of no consequence due to us handling it on a code level could lead to an alter table that runs for days on an older system with a large log table for example.

Reply:
Totally get it.
I find it sad that we are not providing the query straight

Reply:
Totally get it.
I find it sad that we are not providing the query despite the fact that we already have it.
For ``NULL/NOT NULL`` I don't see consequences but for other case like precision or collation. I think it would make sense.
Anyway. If you are against, I'll just remove that part of the code. No big deal

Reply:
We could provide it, especially if it's not executed through the application directly but simply provided as a command that admins can copy paste i don't mind either way